### DISCUSSION TITLE
Does the virtual cache support Redis monitoring? It uses (spring data redis)

### DISCUSSION THREAD

**Discussion by FootyMan** [apache/skywalking#11227] [Q&A]
Does the virtual cache support Redis monitoring? It uses (spring data redis)

**Comment by wu-sheng** [comment]
If it is traced, it should be.

**Reply by FootyMan** [reply]
Do you need the Lettuce plugin？

**Reply by wu-sheng** [reply]
What do you mean by `do you need`?

**Reply by FootyMan** [reply]
<img width="1511" alt="image" src="https://github.com/apache/skywalking/assets/9363830/d14dd895-d7bc-49ea-9330-69b6dbf9f2e8">

I want to display Redis metrics and slow commands

**Reply by FootyMan** [reply]
@wu-sheng support？

**Reply by wu-sheng** [reply]
The docs are there. I am not your service support.
